users/utils.py

```diff
@@ -0,0 +1,25 @@
+import jwt
+
+from django.http            import JsonResponse
+from django.core.exceptions import ObjectDoesNotExist
+
+from users.models           import User
+from my_settings            import SECRET_KEY
+
+def LoginDecorator(func):
```

Comment:
```suggestion
def login_decorator(func):
```
함수명은 소무자 스네이크 케이스로 작성하는 것이 파이썬의 컨벤션입니다.

Reply:
수정완료!